File: src/test/test_sources/two_greets.rb
module NicePerson
  def self.greet(target)
    "Hi #{target}"
  end
end

module NotSoNicePerson
  def self.greet(target)
    "Move along, #{target}"
  end
end

NicePerson.greet("Haidar")
NotSoNicePerson.greet("Spongebob")

File: src/test/test_sources/two_greets.js
const nicePerson = {
    greet: function(target) {
        return "Hi " + target;
    }
};

const notSoNicePerson = {
    greet: function(target) {
        return "Move along, " + target;
    }
};


nicePerson.greet("Haidar");
notSoNicePerson.greet("Spongebob");
